### PR TITLE
Update preview and fix documentation symbols

### PR DESCRIPTION
**Pull request by zanieb**
I don't love the sunrise emoji and 🧪 seems nice :)

Requires #7195

### CODE REVIEW

**Comment by tdulcet** [comment]
While you are at it, maybe also fix the "Hammer and Wrench emoji" to use the actual emoji character. It is currently missing the required variation selector, so it renders as a black and white symbol instead of as a color emoji:
```
🛠 -> 🛠️
```

File: crates/ruff_dev/src/generate_rules_table.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ use ruff_diagnostics::AutofixKind;
 use ruff_workspace::options::Options;
 
 const FIX_SYMBOL: &str = "🛠";
```
Comment (konstin) [inline crates/ruff_dev/src/generate_rules_table.rs:13]: this would be @tdulcet's suggestion:

```suggestion
const FIX_SYMBOL: &str = "🛠️";
```

**Comment by konstin** [comment]
Context for emoji variant selectors: https://www.codejam.info/2021/11/emoji-variation-selector.html. Debugging tool: https://qaz.wtf/u/show.cgi. For me (firefox on ubuntu), it doesn't actually display black and white but just a different emoji:
image

I've checked that hammer (🛠) is in https://www.unicode.org/Public/emoji/5.0/emoji-variation-sequences.txt while construction sign (🚧) isn't.

**Comment by tdulcet** [comment]
> For me (firefox on ubuntu), it doesn't actually display black and white but just a different emoji:

Thanks for providing all that information. For reference, this is what I see with Firefox on Windows 11:
image

**Comment by zanieb** [comment]
They're hilariously identical on Firefox / macOS. I can update it though :)

**Comment by MichaReiser** [comment]
Wdyt of a flask icon (I haven't searched emojis yet) https://fontawesome.com/search?q=experimental&o=r I think it captures the experimental nature better than under construction (because preview features are functional features that need fine tuning, not unfinished work, which is what I associate with the construction emoji)

**Comment by tdulcet** [comment]
> Wdyt of a flask icon (I haven't searched emojis yet)

It looks like the closest emoji would be the Test Tube: 🧪 or Alembic: ⚗️ (the latter would also require a variation selector).

**Comment by zanieb** [comment]
A flask would be nice. I don't like the test tube as much but it might be as close as we can get. I agree they seem better than the construction sign.

**Comment by konstin** [comment]
it should obviously be an eppendorf tube rather than a test tube :P but i think either will do fine

**Review by charliermarsh — APPROVED** [review]

**Review by charliermarsh — APPROVED** [review]

**Pull request merged**